### PR TITLE
Added a setter for class attr to handle svg elements where className cann…

### LINKED ISSUE #2015: svgEl.className is readOnly so live binding does not work
SVG elements class name is readOnly so the only way to set it is to use
svgEl.setAttribute( "class", "..." );

This means that this:
&lt;svg class="{{#if cond}}liveBoundClassName{{/if}}">&lt;/svg>
does not work.

### PR DESCRIPTION
**Pull request by Alfredo-Delgado**
…ot be written to like `svg`. Resolves #2015

### CODE REVIEW

**Comment by justinbmeyer** [comment]
This shouldn't be reading className. I think it should just detect if the element is an svg element and set the attribute.

**Comment by justinbmeyer** [comment]
Also, I think if someone set a className to `{toString: function(){ return "something"}}`, this would fail.

**Comment by Alfredo-Delgado** [comment]
- I added tests for `{toString: function(){ return "something"}}` and they pass.
- The criterium for using `setAttribute` was updated to only apply to `SVGElement` instances.

File: util/attr/attr.js [hunk 1]
```diff
@@ -38,7 +38,18 @@ steal("can/util/can.js", function (can) {
 			 * `element.getAttribute` and `element.setAttribute`.
 			 */
 			map: {
-				"class": "className",
+				"class": function(el, val) {
+					val = val || '';
+
+					if(el instanceof SVGElement) {
```
Comment (Alfredo-Delgado) [inline util/attr/attr.js]: Thinking of IE8, this won't work. I'll update to `if(el.namespaceURI === 'http://www.w3.org/2000/svg')` like [modernizr](https://gist.github.com/mstalfoort/1293822#file-gistfile1-html-L45).

Comment (Alfredo-Delgado) [inline util/attr/attr.js]: Also: http://jsperf.com/instanceof-vs-namespaceuri

**Pull request merged**